modules/combined/doc/content/modules/combined/examples/stm_thermomechanics.md

```diff
@@ -0,0 +1,201 @@
+| Outer Thermal Conductivity | $k_2$ | Uncertain | W/m$\cdot$K |
+| Inner Young's Modulus | $Y_1$ | Uncertain | Pa |
+| Outer Young's Modulus | $Y_2$ | Uncertain | Pa |
+| Inner Poisson's Ratio | $\nu_1$ | Uncertain |  |
+| Outer Poisson's Ratio | $\nu_2$ | Uncertain |  |
+| Inner Thermal Expansion Coef. | $\alpha_1$ | Uncertain | 1/K |
+| Outer Thermal Expansion Coef. | $\alpha_2$ | Uncertain | 1/K |
+| At Rest Temperature | $T_0$ | 300 | K |
+
+!row!
+
+!media combined/temp.png caption=Temperature (K) id=fig:sol_temp style=width:33%;float:left
+
+!media combined/disp.png caption=Displacement (m) id=fig:sol_disp style=width:33%;float:left
+
+!media combined/dispx.png caption=X-Displacement (m) id=fig:sol_dispx style=width:33%;float:left
+
+!media combined/dispy.png caption=Y-Displacement (m) id=fig:sol_dispy style=width:33%;float:left
+
+!media combined/dispz.png caption=Z-Displacement (m) id=fig:sol_dispz style=width:33%;float:left
+
+!row-end!
+
+!listing examples/stochastic/graphite_ring_thermomechanics.i caption=Thermomechanics model input file id=list:thermo
+
+### Uncertain Parameters
+
+A total of nine properties of the model are not known exactly, but with some known probability of values occurring. The probabilities are represented by each parameter's probability density function. All parameters have a independent uniform distribution, $\mathcal{U}(a,b)$, where $a$ and $b$ are the lower and upper limit values for the property, respectively. [tab:uprop] lists the details of each of the property's distribution.
+
+!table caption=Uniform distribution parameters for uncertain properties id=tab:uprop
+| Index | Property | $a$ | $b$ |
+| :- | :- | - | - |
+| 1 | $k_1$ | 20 | 30 |
+| 2 | $k_2$ | 90 | 110 |
+| 3 | $Q_t$ | 9000 | 11000  |
+| 4 | $\alpha_1$ | 1.0$\times10^{-6}$ | 3.0$\times10^{-6}$ |
+| 5 | $\alpha_2$ | 0.5$\times10^{-6}$ | 1.5$\times10^{-6}$ |
+| 6 | $Y_1$ | 2.0$\times10^5$ | 2.2$\times10^5$ |
+| 7 | $Y_2$ | 3.0$\times10^5$ | 3.2$\times10^5$ |
+| 8 | $\nu_1$ | 0.29 | 0.31 |
+| 9 | $\nu_2$ | 0.19 | 0.21 |
+
+### Quantities of Interest
+
+There are a total of ten QoIs for the model, which involve temperature and displacement:
+
+1. Temperature at center of inner surface -- $T_{1,c}$
+1. Temperature at center of outer surface -- $T_{2,c}$
+1. Temperature at end of inner surface -- $T_{1,e}$
+1. Temperature at end of outer surface -- $T_{2,e}$
+1. x-displacement at center of inner surface -- $\delta_{x,1,c}$
+1. x-displacement at center of outer surface -- $\delta_{x,2,c}$
+1. x-displacement at end of inner surface -- $\delta_{x,1,e}$
+1. x-displacement at end of outer surface -- $\delta_{x,2,e}$
+1. z-displacement at end of inner surface -- $\delta_{z,1}$
+1. z-displacement at end of outer surface -- $\delta_{z,2}$
+
+
+[fig:qoi] shows geometrically where these QoI's are located in the model.
+
+!media combined/qoi.png caption=Geometric description of quantities of interest id=fig:qoi style=width:75%
+
+## Results
```

Comment:
It isn't clear that you are preforming two analysis that are computing that same thing. Correct, you are doing the full-order model 100,000 times. You are performing the same calculations with 7,344 runs with PC? The purpose is to show that PC safes a lot of time, correct?

Reply:
I hope my changes make this more clear now